code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -287,14 +294,22 @@
 				if(!E)
 					var/obj/structure/spider/eggcluster/C = new /obj/structure/spider/eggcluster(src.loc)
 					if(ckey)
-						C.player_spiders = 1
+						C.player_spiders = TRUE
+					C.directive = directive
 					C.poison_type = poison_type
 					C.poison_per_bite = poison_per_bite
 					C.faction = faction.Copy()
 					fed--
 		busy = SPIDER_IDLE
 		stop_automated_movement = FALSE
 
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/verb/SetDirective()
+	set name = "Set Directive"
+	set category = "Spider"
+	set desc = "Set a directive for your children to follow"

```

Comment:
Do you think you can do this without using a verb please? Maybe an action button instead?

Reply:
Might as well set up action buttons for the other stuff then

Reply:
I can do that for you! I love spiders n shit

Reply:
but the directive is on you, you action button that pal.

Reply:
already done don't worry